.github/workflows/community.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  issues:
+    types: [ opened ]
+  issue_comment:
+    types: [ created ]
+  pull_request:
+    types: [ opened ]
+  pull_request_review:
+    types: [ submitted ]
+  pull_request_review_comment:
+    types: [ created ]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
```

Comment:
If what @BOLT04 and I discussed is true then this will also fail.

But this appears to be using this GitHub action: https://github.com/actions/first-interaction

```yaml
      - uses: actions/checkout@v1
      - uses: EddieJaoudeCommunity/gh-action-community/src/welcome@main
```

Reply:
Thanks @AllanRegush , you are right, it needs to be like the example in the Issue #20 https://github.com/EddieJaoudeCommunity/support/blob/main/.github/workflows/community.yml